extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetInstructions.java

```diff
@@ -445,7 +445,8 @@ public static class Builder {
         private int maximumDictionaryKeys = defaultMaximumDictionaryKeys;
         private int maximumDictionarySize = defaultMaximumDictionarySize;
         private boolean isLegacyParquet;
-        private int targetPageSize = defaultTargetPageSize;
+        @VisibleForTesting
+        int targetPageSize = defaultTargetPageSize;
```

Comment:
I see why you did this, but it's a little unfortunate. Is the default inappropriately high? Should we make it configurable?